website/docs/docs/dbt-cloud/access-control/access-control-overview.md

```diff
@@ -0,0 +1,120 @@
+
+Role-based permissions are applied to _groups_ and pertain to _projects_. The
+assignable permissions themselves are granted via _permission sets_.
+
+
+### Groups
+
+A group is a collection of users. Users may belong to multiple groups. Members
+of a group inherit any permissions applied to the group itself.
+
+Users can be added to a dbt Cloud group based on their group memberships in the
+configured [Identity Provider](sso-overview) for the account. In this way, dbt
+Cloud administrators can manage access to dbt Cloud resources via identity
+management software like Azure AD, Okta, or GSuite. See _SSO Mappings_ below for
+more information.
+
+You can view the groups in your account or create new groups from the **Team > Groups**
+page in your Account Settings.
+
+<Lightbox
+    src="/img/docs/dbt-cloud/dbt-cloud-enterprise/access-control/group-list.png"
+    title="Viewing a list of groups in the Account Settings page."
+/>
+
+
+### SSO Mappings
+
+SSO Mappings connect Identity Provider group membership to dbt Cloud group
+membership. SSO Mappings are declared as a list of IdP group names within a
+specific dbt Cloud group. When a user logs into dbt Cloud via a supported
+identity provider, their IdP group memberships are synced with dbt Cloud. Upon
+logging in successfully, the user's group memberships (and therefore, permissions)
+are adjusted as needed within dbt Cloud automatically.
+
+:::tip Creating SSO Mappings
+
+While dbt Cloud supports mapping multiple IdP groups to a single dbt Cloud
+group, we recommend using a 1:1 mapping to make administration as simple as
+possible. Consider using the same name for your dbt Cloud groups and your IdP
+groups.
+
+:::
+
+<Lightbox
+    src="/img/docs/dbt-cloud/dbt-cloud-enterprise/access-control/group-detail.png"
+    title="Users in the 'data-team-dbt-analysts' or 'data-team-dbt-admins' groups will be assigned to the Analysts dbt Cloud group when they log into dbt Cloud"
+/>
+
+### Permission Sets
+
+Permission sets are predefined collections of granular permissions. Permission
+sets combine low-level permission grants into high-level roles that can be
+assigned to groups. Some examples of existing permission sets are:
+ - Account Admin
+ - Git Admin
+ - Job Admin
+ - Job Viewer
+ - ...and more
+
+For a full list of enterprise permission sets, see [Enterprise Permissions](/docs/dbt-cloud/access-control/enterprise-permissions).
+These permission sets are available for assignment to groups and control the ability
+for users in these groups to take specific actions in the dbt Cloud application.
+
+In the following example, the _Analysts_ group is configured with the
+**Analyst** permission set on _All Projects_ and the **Job Admin** permission
+set on the _BQ test_ project.
+
+<Lightbox
+    src="/img/docs/dbt-cloud/dbt-cloud-enterprise/access-control/group-permissions.png"
+    title="Configuring permissions for the Analysts group"
+/>
+
+
+## FAQs
```

Comment:
Also common question: Can I force a user to logout in order to resync their permissions?